### PR TITLE
updated structure to match template

### PR DESCRIPTION
**Pull request by CrystalOnScript**

### CODE REVIEW

**Comment by jpettitt** [comment]
@kushal @dackmin please take a look since you contributed to these files.

**Comment by dackmin** [comment]
lgtm

File: extensions/amp-access-scroll/amp-access-scroll.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ $category@: dynamic-content
 formats:
   - websites
 teaser:
-  text: Integrates wth Scroll, a better internet that doesn't rely on ads.
+  text: Integrates with Scroll paywall.
```
Comment (kushal) [inline extensions/amp-access-scroll/amp-access-scroll.md:6]: Scroll isn't really a "paywall" — can we call this "membership"?

Comment (CrystalOnScript) [reply extensions/amp-access-scroll/amp-access-scroll.md]: Done!

**Pull request merged**